configure.ac

```diff
@@ -0,0 +1,76 @@
+dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
+AC_PREREQ([2.60])
+define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_COPYRIGHT_YEAR, 2015)
+AC_INIT([Armory],m4_esyscmd([grep "^BTCARMORY_VERSION" armoryengine/ArmoryUtils.py | cut -d "(" -f2 | cut -d ")" -f1 | tr -d '[[:space:]]' | tr ',' '.']))
+AC_CONFIG_SRCDIR([cppForSwig/BlockObj.cpp])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_MACRO_DIR([build-aux/m4])
+
+AC_CANONICAL_HOST
+
+dnl faketime breaks configure and is only needed for make. Disable it here.
+unset FAKETIME
+
+case $host in
+    *mingw*)
+        TARGET_OS=windows
+        ;;
+    *darwin*)
+        TARGET_OS=darwin
+        ;;
+    *linux*)
+        TARGET_OS=linux
+        ;;
+    *)
+        ;;
+esac
+
+dnl Automake init set-up and checks
+AM_INIT_AUTOMAKE([no-define subdir-objects foreign])
+
+# Enable debug
+AC_ARG_ENABLE([debug],
+    [AS_HELP_STRING([--enable-debug],
+                    [use debug compiler flags and macros (default is no)])],
+    [enable_debug=$enableval],
+    [enable_debug=no])
+
+dnl Define variables
+AC_ARG_VAR([PYVER], [Defines the version of Python to use (e.g. python2.7)])
```

Comment:
The comment doesn't seem to be totally accurate. It looks like we just use the variable to choose between "python" and "python3". That said, it might not hurt to insert a check for the version of Python we're using. (Py2.6 was the minimum but it went to 2.7 awhile back, and I don't think we were ever able to restore 2.6 support.) https://github.com/coccinelle/coccinelle/blob/master/setup/cocci.m4 seems to have an example of how to get the exact version.